src/mutations/events.ts

```diff
@@ -13,36 +14,51 @@ const createEvent = async (parent, data, ctx): Promise<Event> => {
   const image_base64: string = data.image_base64;
   const description: string = data.description;
 
-  //image_url should be a link pointing to S3
-  //a utility function to be further implemented
-  const image_url = image_base64;
-
-  const event: Event = await ctx.prisma.createEvent({
+  let event: Event = await ctx.prisma.createEvent({
     title,
     start,
     end,
     venue,
-    image_url,
     description,
+    image_url: "",
     organiser: {
       connect: {
         id: user.id,
       },
     },
   });
 
+  //Store image_base64 to S3 bucket with filename <event.id>
+  //And retrieve image_url back to be stored in the database
+  const { id }: { id: string } = event;
```

Comment:
not sure if type defs `id: string` is necessary since event is of type `Event` declared earlier which has `id: string` already

Reply:
ahh true that, I'll modify this in the next commit.